scss/civicrm/full-text-search/_full-text-search.scss

```diff
@@ -0,0 +1,19 @@
+.CRM_Contact_Form_Search_Custom .crm-search-form-block table.form-layout-compressed td {
+  padding: 2px 0;
+  width: auto;
```

Comment:
I think this is for td.label only to control width? if that's the case set it under https://github.com/civicrm/org.civicrm.shoreditch/pull/93/files#diff-21377b9c511c84c1f03c6dd1774b2ce3R11